.github/workflows/main.yml

```diff
@@ -568,9 +568,41 @@ jobs:
         run: |
           set -x
           sudo rm -rf "/usr/local/share/boost" "$AGENT_TOOLSDIRECTORY" "/usr/local/lib/android" "/usr/share/dotnet"
-          docker run --rm -d --name indypool --network host $DOCKER_IMG_NAME_POOL
-          docker run --rm -d --name postgres --network host -e POSTGRES_PASSWORD=mysecretpassword postgres:12.1
+          docker run --rm -d --name indypool  --network host $DOCKER_IMG_NAME_POOL
+          docker run --rm -d --name mysql     --network host -e MYSQL_ROOT_PASSWORD=mysecretpassword mysql:5.7.35
+
+      - name: Create Wallet DB schema, run migrations
+        run: |
+          set -x
```

Comment:
I don't think `set -x` is necessary when running single command (applies everywhere in this PR)

Reply:
`set -x` can be useful even if just a single command is to be run, as it gives you ability to see what bash variables been interpolated

Reply:
The UI shows the executed code and defined environment variables anyway.